src/encoderWorker.js

```diff
@@ -70,7 +70,9 @@ var OggOpusEncoder = function( config, Module ){
     numberOfChannels: 1,
     originalSampleRate: 44100,
     resampleQuality: 3, // Value between 0 and 10 inclusive. 10 being highest quality.
-    serial: Math.floor(Math.random() * 4294967296)
+    serial: Math.floor(Math.random() * 4294967296),
+    streamOpusPackets: true,
+    framesPerCallback: 1 // Value between 1 and 2 inclusive. Only used when streamOpusPackets is true.
```

Comment:
is it possible to allow this to be higher?  if not, then i'd make it a boolean value (`cacheFrameForCallback`), but if so then i'd generalize the code below to use the config value and return a variable length array

Reply:
I'll make it a boolean. Making it variable would be slightly harder, and not actually used by us, so I thought it wasn't worth it. If we ever made a PR back to the original repository, I'd want to have the Opus solution anyway.